app/controllers/coronavirus_form/data_export_checkbox_controller.rb

```diff
@@ -0,0 +1,92 @@
+require "csv"
+
+class CoronavirusForm::DataExportCheckboxController < ApplicationController
+  if ENV.key?("DATA_EXPORT_BASIC_AUTH_USERNAME") && ENV.key?("DATA_EXPORT_BASIC_AUTH_PASSWORD")
+    http_basic_authenticate_with(
+      name: ENV.fetch("DATA_EXPORT_BASIC_AUTH_USERNAME"),
+      password: ENV.fetch("DATA_EXPORT_BASIC_AUTH_PASSWORD"),
+    )
+  end
+
+  def show
+    respond_to do |format|
+      format.html
+      format.csv do
+        render csv: produce_csv(usage_statistics(params[:start_date], params[:end_date])),
+               filename: "data-export"
+      end
+    end
+  end
+
+  def checkbox_options
+    options = I18n.t("coronavirus_form.groups")[:filter_questions][:questions][:need_help_with][:options]
+
+    options.index_by do |_option|
+      :need_help_with
```

Comment:
In the form, this value is saved as "#{option.parameterize.underscore}". Should the same value be used here?